File: user_tests/bisect/bisect.py
def bisect_right(data: List[int], item: int) -> int:
    low = 0
    high: int = int(len(data))
    while low < high:
        middle = int((low + high) / 2)
        if item < data[middle]:
            high = middle
        else:
            low = middle + 1
    return low

File: user_tests/bisect/bisect.jl
function bisect_right(data::Vector{Int64}, item::Int64)::Int64
    low = 0
    high::Int64 = Int(length(data))
    while low < high
        middle = Int(floor((low + high) / 2))
        if item < data[middle+1]
            high = middle
        else
            low = middle + 1
        end
    end
    return low
end